src/EventListener/ResponseSecurer.php

```diff
@@ -6,21 +6,35 @@
 
 class ResponseSecurer
 {
+    private bool $isDebug;
+
+    public function __construct(bool $isDebug)
+    {
+        $this->isDebug = $isDebug;
+    }
+
     /**
      * Add some headers to the response to make our application more secure
-     * see https://www.owasp.org/index.php/List_of_useful_HTTP_headers
+     * see https://owasp.org/www-project-secure-headers
      *
      * @param ResponseEvent $event
      */
     public function onKernelResponse(ResponseEvent $event)
     {
-        // provides clickjacking protection
-        $event->getResponse()->headers->set('X-Frame-Options', 'deny');
-
-        // enables the XSS filter built into most recent browsers
-        $event->getResponse()->headers->set('X-XSS-Protection', '1; mode=block');
+        /*
+         * We only send the security headers when we're not in dev mode
+         */
+        if (!$this->isDebug) {
+            $event->getResponse()->headers->set('Content-Security-Policy',
+                "default-src 'self';" . // Default rule: only allow content from our own domain
+                "style-src 'self' https://fonts.googleapis.com;" . // Allow Google Fonts
+                "font-src 'self' https://fonts.gstatic.com;" . // Allow Google Fonts
+                "frame-src 'none';" . // Block all iframes
```

Comment:
Dus alle embeds gaan ook niet werken? 
Voor framework zie ik niet direct een issue

Reply:
Iframes gaan niet ingeladen worden, inline scripts gaan niet uitgevoerd worden, externe CSS & JS gaan niet ingeladen worden, etc..

Per project kan de header uitgebreid worden met URLs die wel toegelaten zijn, zoals ik hier al by default voor Google Fonts heb gedaan. Ikzal nog een doc pagina schrijven om dit ter verduidelijken, want zal voor een paar niche features in sommige projecten wel breaking zijn in de toekomst.

Reply:
Added docs 
a6a8b46